rush.json

```diff
@@ -18,6 +18,11 @@
     "defaultBranch": "master"
   },
   "projects": [
+    {
+      "packageName": "@itwin/breakdown-trees-react",

```

Comment:
change `version-policies.json` and bump the pkg's minor and set it to be a pre-release

Reply:
changed